recipes/codem/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "codem" %}
+{% set version = "0.23.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/codem-{{ version }}.tar.gz
+  sha256: 21f609590b11d6e552ecf62c7872bc200f3e6b2e191eb0527e9a55ee512f8950
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - codem = codem.main:main
+    - vcd = vcd.main:main 
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
```

This is in the pkg metadata so it is nice to have it here too. You are probably already getting that version anyway.

```suggestion
    - pip
    - setuptools >=64.0
```

---

Fixed  in fbdd004